### ISSUE TITLE
Too many slashes while decorating

### ISSUE THREAD

**Issue by greengreens** [nikhilkalige/docblockr#281]
Small issue here, and if I get time I will look into it myself.

As you can see in the GIF below, when using the decorate function, too many slashes are created (seems to always be +2 too many). Language tested is Typescript if that helps.

Possibly an incorrect offset when counting decoration length?

![docblockr-comment](https://user-images.githubusercontent.com/6889815/28835974-2edefb3a-76df-11e7-9685-0bc90c723bf4.gif)

**Comment by MoritzKn** [comment]
Good catch and thanks for the report! If you would like to fix it your self, that would be perfect. Feel free to ask if you run in any problems? Through I haven't looked at this part of the code base for quite some time...
